docs/en/interfaces/formats.md

```diff
@@ -2134,6 +2134,8 @@ To exchange data with Hadoop, you can use [HDFS table engine](/docs/en/engines/t
 - [input_format_parquet_case_insensitive_column_matching](/docs/en/operations/settings/settings-formats.md/#input_format_parquet_case_insensitive_column_matching) - ignore case when matching Parquet columns with ClickHouse columns. Default value - `false`.
 - [input_format_parquet_allow_missing_columns](/docs/en/operations/settings/settings-formats.md/#input_format_parquet_allow_missing_columns) - allow missing columns while reading Parquet data. Default value - `false`.
 - [input_format_parquet_skip_columns_with_unsupported_types_in_schema_inference](/docs/en/operations/settings/settings-formats.md/#input_format_parquet_skip_columns_with_unsupported_types_in_schema_inference) - allow skipping columns with unsupported types while schema inference for Parquet format. Default value - `false`.
+- [input_format_parquet_min_read_batch_size_rows](/docs/en/operations/settings/settings-formats.md/#input_format_parquet_min_read_batch_size_rows) - minimum number of rows to read in one thread in Parquet format. Default value - `100000`.
+- [input_format_parquet_min_read_batch_size_rows](/docs/en/operations/settings/settings-formats.md/#input_format_parquet_min_read_batch_size_rows) - minimum number of bytes (uncompressed) to read in one thread in Parquet format. Default value - `25'600'000`.
```

Comment:
Any suggestions about better defaults are welcome. Also not sure if we need both settings, but I decided to do similar to our `min_insert_block_size_rows/min_insert_block_size_bytes`